cg/models/demultiplex/flow_cell.py

```diff
@@ -220,6 +216,16 @@ def validate_sample_sheet(self) -> bool:
         except (SampleSheetError, ValidationError) as error:
             LOG.warning("Invalid sample sheet")
             LOG.warning(error)
+            if self.bcl_converter == BclConverter.BCL2FASTQ:
```

Comment:
LOG.warning (SampleSheetHeader.get(self.bcl_convert))

Would this work if we have a constant?

Reply:
I have some recollection that you would need to import advanced enums (aenum) to get this working and that a more common approach is to create a dict with the converters as keys and the headers as values.

Reply:
It might work if you make the enum string based. I am using this in Trailblazer:

```
class TrailblazerStatus(str, Enum):
    """Trailblazer allowed status."""

    CANCELLED: str = "canceled"
    COMPLETED: str = "completed"
    COMPLETING: str = "completing"
    ERROR: str = "error"
    FAILED: str = "failed"
    PENDING: str = "pending"
    QC: str = "qc"
    RUNNING: str = "running"


TrailblazerStatus[single_status.upper()]
```
Here: https://github.com/Clinical-Genomics/trailblazer/blob/master/trailblazer/apps/slurm/api.py